roles/nginx/templates/vhost.j2

```diff
@@ -39,6 +39,8 @@ server {
 
 	root /var/www/georchestra/htdocs/;
 
+  client_max_body_size 100M;
```

Comment:
weird indent ?

Reply:
fixed ; my text editor ignored the fact that the file was using a mix of spaces and tabs.